### ISSUE TITLE
[Usable – Make Code – Home page] There is no chevron or underline for 'Use Developer token' link under 'Sign in with GitHub' pop-up window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3583]
**User Experience:**
User will be misguided If any actionable control(link) doesn't have underline or chevron as they will skip the functionality of such link. Not only this, user will face disappointment if visually appearing link display as text as user won't be proceed further for relevant action.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to "Your GitHub Repo" option and navigate through the all the controls of dialog "Sign in with GitHub"..
6. Verify visual 'Use Developer token' link by using Developer tool(F12).

**Actual Result:**
There is no chevron or underline for 'Use Developer token' link under 'Sign in with GitHub' pop-up window and due to this, user is getting affected as he/she might consider that link as text and will skip the functionality of such link.

**Expected Result:**
There must be chevron or underline on visual 'Use Developer token' link under 'Sign in with GitHub' pop-up window, so that user could easily navigate that link to get more details.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

![Usable_There is no underline or chevron on highlighted 'Use Developer Token' link under 'Sign in with GitHub' pop-up window](https://user-images.githubusercontent.com/72871089/98104627-b38d5100-1ebc-11eb-831a-6fe9c0d58bcd.jpg)

**Comment by jwunderl** [comment]
Styled as a link
